src/rngs/mock.rs

```diff
@@ -13,12 +13,23 @@ use rand_core::{impls, Error, RngCore};
 #[cfg(feature = "serde1")]
 use serde::{Serialize, Deserialize};
 
-/// A simple implementation of `RngCore` for testing purposes.
+/// A mock generator yielding very predictable output
 ///
 /// This generates an arithmetic sequence (i.e. adds a constant each step)
 /// over a `u64` number, using wrapping arithmetic. If the increment is 0
 /// the generator yields a constant.
 ///
+/// Other integer types (64-bit and smaller) are produced via cast from `u64`.
+///
+/// Other types are produced via their implementation of [`Rng`](crate::Rng) or
+/// [`Distribution`](crate::distributions::Distribution).
+/// Output values may not be intuitive and may change in future releases but
+/// are considered
+/// [portable](https://rust-random.github.io/book/portability.html).
+/// (`bool` output is true when bit `1u64 << 31` is set.)
```

Comment:
just want to be sure, it is `1u64 << 31` for any endianness / arch ?

Reply:
Yes.